src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -105,17 +105,19 @@ export default class ArrayField extends React.Component {
   }
 
   scrollToRow(id) {
-    setTimeout(() => {
-      scroller.scrollTo(
-        `table_${id}`,
-        window.Forms?.scroll || {
-          duration: 500,
-          delay: 0,
-          smooth: true,
-          offset: 0,
-        },
-      );
-    }, 100);
+    if (!this.props.uiSchema['ui:options'].doNotScrollOnAdd) {
```

Comment:
Let's move this to a separate PR so we can get the rest of the app approved. Thanks

Reply:
Sure thing!